src/api/openweathermap.ts

```diff
@@ -15,9 +15,9 @@ type OpenWeatherApiResponse = {
 };
 
 export type WeatherInfo = {
-  temperatureCelsius: number;
```

Comment:
do not remove, may be useful in the future, put ?

Reply:
https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019